src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -457,7 +458,7 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
             [this](const std::shared_ptr<const ngraph::Node> &node) -> bool {
                 auto sub_graph_op = std::dynamic_pointer_cast<const ngraph::op::util::SubGraphOp>(node);
                 int64_t num_iter = sub_graph_op->get_num_iterations();
-                if (num_iter == 1) {
+                if (num_iter < 16) {
```

Comment:
I think default value for enable_loop_unrolling should be still true, and this condition should allow to disable loop unrolling even for num_iter < 16. 

```cpp
// If unrolling is disabled, then unroll only for single iteration
if (!config.enable_loop_unrolling)
    return num_iter != 1;

// Otherwise, unroll when iters < 16
return num_iter >= 16;


```

Reply:
Fixed